app/views/categories/index.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="container mt-4">
+  <div class="nav-header">
+    <%= button_to "Sign Out", destroy_user_session_path, method: :delete, class: "btn btn-danger signout-btn border-0", style: "background-color: #3778c2;" %>
+    <h2>Categories </h2>
+  </div>
+  <% @categories.each do |category| %>
+    <div class="category-item border p-3 mb-3">
+      <div class="d-flex justify-content-between align-items-center">
+        <div class="d-flex align-items-center">
+          <img src="<%= category.icon %>" alt="Category Icon" class="img-fluid img-avatar" width='50px'/>
+          <h2 class="ms-5 md-3"><%= category.name %></h2>
+        </div>
+        <p class="mb-0">
+          $ <%= category.expenses.sum(:amount) %>
+        </p>
+      </div>
+      <small><%= category.created_at.strftime('%d-%b-%Y') %></small>
+      <%= link_to 'Transactions', category_path(category), class: "text-text-white ms-3" %>
+    </div>
+  <% end %>
```

Comment:
- [ ] The same happens with your categories view:

|Template|Your design|
|---|---|
| ![Screenshot from 2023-08-18 21-24-48](https://github.com/eudondian/Budget-App/assets/77069011/597fec4f-3e4e-4781-8920-f15daa01dcbb) | ![image](https://github.com/eudondian/Budget-App/assets/77069011/6249528b-585a-4ba1-a475-14544e4b7317) |

Please consider removing the blank space at the top, the horizontal margin, and moving the button all the way to the bottom of the page! 🤓 🔧